sklearn/model_selection/tests/test_search.py

```diff
@@ -641,6 +641,69 @@ def test_refit():
     clf.fit(X, y)
 
 
+def test_refit_callable():
+    """
+    Test refit=callable, which adds flexibility in identifying the
+    "best" estimator.
+    """
+    def refit_callable(cv_results):
+        """
+        A dummy function tests `refit=callable` interface.
+        Return the index of a model that has the least
+        `mean_test_score`.
+        """
+        # Fit a dummy clf with `refit=True` to get a list of keys in
+        # clf.cv_results_.
+        X, y = make_classification(n_samples=100, n_features=4,
+                                   random_state=42)
+        clf = GridSearchCV(LinearSVC(random_state=42), {'C': [0.01, 0.1, 1]},
```

Comment:
This test case seems to not have anything to do with the callable?

Reply:
This dummy clf helps with getting a list of keys in clf.cv_results, and later we can assert that all those keys exist in `cv_results_` after we call `refit`

Reply:
There is a comment above to explain